spec/lib/capsule_crm/history_spec.rb

```diff
@@ -301,7 +301,7 @@
 
     it 'has the correct history item entry date' do
       expect(subject['historyItem']['entryDate']).
-        to eql(history.entry_date.strftime("%Y-%m-%dT%H:%M:%SZ"))
+        to eql(history.entry_date.utc.strftime("%Y-%m-%dT%H:%M:%SZ"))
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.